addon/async-test-waiter.ts

```diff
@@ -1,4 +1,4 @@
-import { ITestWaiter, ITestWaiterDebugInfo, Primitive, WaiterName } from './types';
+import { Primitive, TestWaiter, TestWaiterDebugInfo, WaiterName } from './types';
```

Comment:
Thoughts on moving this implementation directly into `addon/build-waiter.ts`, and restructuring the class to be a non-default (possibly underscored) export?

Reply:
Ya I like this. I'll do it with the other changes.